### DISCUSSION TITLE
App template with signup/login/logout

### DISCUSSION THREAD

**Discussion by jamesyoung1337** [quasarframework/quasar#10512] [CLI - Capacitor / Cordova modes]
Hi,

Quasar looks very promising.  I've used Cordova and Capacitor before, so it looks and feels somewhat familiar.

I'm looking for an example app that does something like this:

1. Checks for a login token, and if found and its valid, take user to main screen with menu (e.g. drawer menu on the side, profile... etc).
2.  If no token, show the login/signup page.
3. On successful signup go back to login screen.
4. On login, show the main screen as in 1).

It would be nice to have an onboarding process, like a few full screen slides or something like that, maybe even a set up wizard with steps.

I have not yet found an easy to develop process to kick off an app with these features.

Does anyone know of any app templates with a signup/signin screen, which then navigates to main screen when successfully logged in?

Thanks!

**Comment by ontwikkelfabriek** [comment]
@jamesyoung-info 

You could take a look here:
https://github.com/quasarframework/quasar-awesome

Here's an Qv1 app with login etc:
https://github.com/cartovarc/github-method

Here's a auth plugin for Qv1:
https://github.com/stefanvanherwijnen/quasar-app-extension-http-authentication
